test/solver.jl

```diff
@@ -189,6 +189,20 @@ k = 1
         @test h_S ≈ svdvals(A)
         @test abs.(h_Vt*F.Vt') ≈ Matrix(one(elty)*I, n, n)
     end
+    
+    @testset "gesvdj!" begin
+        A              = rand(elty,m,n)
+        d_A            = CuArray(A)
+        d_U, d_S, d_V  = CUSOLVER.gesvdj!('V',1,d_A)
+        h_S            = collect(d_S)
+        h_U            = collect(d_U)
+        h_V            = collect(d_V)
+        F              = svd(A, full=true)
+        @test_broken abs.(h_U'h_U) ≈ Matrix(one(elty)*I, m, m) atol=1e-6
```

Comment:
Why is this broken?

Reply:
We should actually make sure to test both the tall and wide case for both SVDs. It's so easy to make small errors with the dimensions.

Reply:
Because the off-diagonal elements are sometimes pretty large

Reply:
Just to check, you mean prepare A with dimensions `(m, n)` and `(n, m)` where `m > n`, right?

Reply:
Yes, exactly.